Cubical/Homotopy/HSpace.agda

```diff
@@ -49,3 +143,59 @@ S1-AssocHSpace : AssocHSpace S1-HSpace
         (λ { base → refl ; (loop i) → refl})
         refl
 μ-assoc-filler S1-AssocHSpace _ _ = refl
+
+-- →∙
+→∙-HSpace : (A : Pointed ℓ) (X : Pointed ℓ') (hX : HSpace X) → HSpace (A →∙ X ∙)
```

Comment:
If you like, you can introduce an anonymous module with the arguments 
```(A : Pointed ℓ) (X : Pointed ℓ') (hX : HSpace X)```
Then you could open hX and have a less noisy pattern matching.

Reply:
Thanks, done in d80a3b4a